server/index.js

```diff
@@ -30,15 +33,71 @@ app.use((req, res, next) => {
       'max-age=86400; includeSubDomains'
     )
   }
+  if (config.env.stage.cspEnabled) {
+    const cdn = 'cdn.algorithmia.com'
+    // No convenient way to hook into these dynamically. If adding an inline
+    // script, a new CSP SHA will have to be added manually here.
+    // Chrome will tell you which SHA it expects if you have any CSP errors,
+    // so you don't have to calculate them yourself.
+    const cspShas = [
+      'WmaB/BZsNpo2j+CMricdhZ2p4mn+Q54VCeUr3ceYLFA='
```

Comment:
:bulb: Mind adding a note which specific script this SHA relates to?

Reply:
Updated!